client/my-sites/invites/invite-accept/index.jsx

```diff
@@ -88,15 +89,35 @@ let InviteAccept = React.createClass( {
 		);
 	},
 
+	clickedNoticeSiteLink() {
+		analytics.tracks.recordEvent( 'calypso_invite_accept_notice_site_link_click' );
+	},
+
 	acceptedNotice( invite ) {
 		invite = invite || this.state.invite;
 		let displayOnNextPage = true;
+		let takeATour = (
```

Comment:
the three variables declared here could be `const` including `displayOnNextPage` that I introduced :blush: